managedplugin/plugin_test.go

```diff
@@ -56,38 +56,38 @@ func TestManagedPluginCloudQuery(t *testing.T) {
 	ctx := context.Background()
 	tmpDir := t.TempDir()
 	cfg := Config{
-		Name:     "test",
+		Name:     "azuredevops",
```

Comment:
The test plugin is private so we can't download it without an auth token. It worked before due to a bug we fixed